ntoskrnl/ke/amd64/trap.S

```diff
@@ -1137,6 +1137,30 @@ KiConvertToGuiThreadFailed:
 
 ENDFUNC
 
+EXTERN KiGetTrapContextInternal:PROC
+
+/*
+ * VOID
+ * KiGetTrapContext(
+ *     _Out_ PKTRAP_FRAME TrapFrame,
+ *     _In_ PCONTEXT Context,
+ *     _In_ KPROCESSOR_MODE RequestorMode);
```

Comment:
```suggestion
 * VOID
 * KiGetTrapContext(
 *     _In_ PKTRAP_FRAME TrapFrame,
 *     _Out_ PCONTEXT Context);
```
This is the prototype as you've added it in amd64/ke.h

Reply:
Fixed